File: nomad-samples/portal/py/movies/content_admin_api/content/delete_tag_or_collection.py
from constants.project_constants import *
from exceptions.api_exception_handler import *

import json, requests

def delete_tag_or_collection(AUTH_TOKEN: str, TYPE: str, CONTENT_ID: str, TAG_ID: str, CONTENT_DEFINITION: str) -> dict:

    # Check for valid parameters
    if (not AUTH_TOKEN):
        raise Exception("Authentication Token: The authentication token is invalid")

    API_URL = ADMIN_API_URL + "/admin/" + TYPE + "/content/delete"
        
    # Create header for the request
    HEADERS = {
  	    "Authorization": "Bearer " + AUTH_TOKEN,
        "Content-Type": "application/json"
    }

    # Build the payload BODY
    BODY = {
        "items": [
            {
                "contentDefinition": CONTENT_DEFINITION,
                "contentId": CONTENT_ID,
                "tagId": TAG_ID
            }
        ]
    }

    try:
        # Send the request
        RESPONSE = requests.post(API_URL, headers= HEADERS, data= json.dumps(BODY))

        if not RESPONSE.ok:
            raise Exception()

        return json.loads(RESPONSE.text)

    except:
        api_exception_handler(RESPONSE, "delete tag or colleciton failed")

File: nomad-samples/admin/js/content-admin-api/content/delete-tag-or-collection.js
import * as prjConstants from "../constants/project-constants.js";
import apiExceptionHandler from "../exceptions/api-exception-handler.js";

export default async function deleteTagOrCollection(AUTH_TOKEN, TYPE, TAG_ID)
{
    const HEADERS = new Headers();
    HEADERS.append("Content-Type", "application/json");
    HEADERS.append("Authorization", `Bearer ${AUTH_TOKEN}`);
    
    // Post
    const RESPONSE = await fetch(`${prjConstants.ADMIN_API_URL}/admin/${TYPE}/${TAG_ID}`, {
        method: "DELETE",
        headers: HEADERS
    }).catch((exception) => {
        throw exception;
    });

    // Check for success
    if (RESPONSE.ok) {
        return;
    }

    // There was an error
    await apiExceptionHandler(RESPONSE, "Delete tag or collection failed");
}